wireless_modules/middle_module/main.py

```diff
@@ -0,0 +1,89 @@
+import machine
+import ubinascii
+import utime
+from mqtt_client import Client
+
+
+try:
+    import config
+except FileNotFoundError:
+    print('Error importing config.py, ensure a local version of config.py exists')
+
+
+# Define module number and MQTT topics to publish to
+MODULE_NUM = "2"
+PUB_DATA_TOPIC = b"/v3/wireless-module/{}/data".format(MODULE_NUM)
+PUB_TOPIC_LOW_BATTERY = b"/v3/wireless-module/{}/low-battery".format(MODULE_NUM)
```

Comment:
Is there any chance we could change these to using the % format similar to `"Connected to %s" % self.mqtt_broker`. Just so that all of the code uses the same formatting style.

Reply:
Is there a preference for using % format, otherwise I think .format() is much easier to read

Reply:
I think `.format()` is easier as well. I just was the % format in other code blocks. Maybe we should change all to `.format()`.